server/service/hosts.go

```diff
@@ -172,11 +178,15 @@ type deleteHostsRequest struct {
 }
 
 type deleteHostsResponse struct {
-	Err error `json:"error,omitempty"`
+	Err        error `json:"error,omitempty"`
+	StatusCode int   `json:"-"`
```

Comment:
Nit: `statusCode int` should also work.

Reply:
Ok. I assume that `Err` should have been `err` in line with the idea of package-level accessibility.